VERSION

```diff
@@ -0,0 +1,3 @@
+VERSION_MAJOR = 0
```

Comment:
Should MAJOR not be 1?

Reply:
Hi @galak
I would prefer to make it in two steps to set the version update in the last patch of the release.